ice.go

```diff
@@ -0,0 +1,274 @@
+package webrtc
+
+import (
+	"github.com/pion/sdp/v2"
+	"github.com/pion/webrtc/v2/internal/ice"
+)
+
+//go:generate go run gen/genaliasdocs.go -- $GOFILE
+
+const (
+
+	// ICETransportPolicyRelay is an alias for ice.ICETransportPolicyRelay
```

Comment:
If the thing we are aliasing has a doc, can we use that instead?

If I am reading the godoc these comments aren't helpful currently.

Reply:
Yea, that's the plan, i just didn't get there yet (TODO: "finish the script for generating docs for the aliases").

On a related note: how should we handle wasm binding aliases?
1. create ice.go for the common aliases, ice_go.go for go only aliases and ice_js.go for wasm aliases
  - if we choose this, where should we put aliases like ICEServer, that are present in both but defined separately for go/js (also, they might have different docs, so in this case which one should we use?)
2. create ice_go.go and ice_js.go only for go/wasm alises

Should I also create a tests for comparing js and wasm aliases?
- for types, if the type is present in both go and wasm, check if it has the same exported methods with the same signature
- for constants, if the constant is present in both go and wasm, check if they have the same value
- for functions, if the function is present in both go and wasm, check if the functions have the same signature